docs/dxp/7.x/en/liferay-internals/extending-liferay/overriding-osgi-services/reconfiguring-components-to-use-your-osgi-service/resources/liferay-m1t1.zip/m1t1-api/src/main/java/com/acme/m1t1/M1T1.java

```diff
@@ -0,0 +1,7 @@
+package com.acme.m1t1;
+
+public interface M1T1 {
+
+	public String doSomething();
+
+}
```

Comment:
For this one, I had:
```java
package com.acme.s1j6;

import aQute.bnd.annotation.ProviderType;

@ProviderType
public interface S1J6 {

	public String doSomething();
}
```

Using `@ProviderType` to define the interface as a capability that components can provide (implement) or consume.